packages/data-context/src/data/ProjectConfigIpc.ts

```diff
@@ -279,86 +277,73 @@ export class ProjectConfigIpc extends EventEmitter {
+     */
+    let tsxLoader = this.nodeVersion && semver.lt(this.nodeVersion, '20.6.0') ? `--loader ${tsx}` : `--import ${tsx}`
+
+    // If they've got TypeScript installed, we can use tsx for CommonJS and ESM.
+    // @see https://tsx.is/dev-api/node-cli#node-js-cli
+    const userHasTypeScriptInstalled = hasTypeScriptInstalled(this.projectRoot)
+
+    if (userHasTypeScriptInstalled) {
       debug('found typescript in %s', this.projectRoot)
-      if (isProjectUsingESModules) {
-        debug(`using --experimental-specifier-resolution=node with --loader ${tsNodeEsm}`)
-        // Use the ts-node/esm loader so they can use TypeScript with `"type": "module".
-        // The loader API is experimental and will change.
-        // The same can be said for the other alternative, esbuild, so this is the
-        // best option that leverages the existing modules we bundle in the binary.
-        // @see ts-node esm loader https://typestrong.org/ts-node/docs/usage/#node-flags-and-other-tools
-        // @see Node.js Loader API https://nodejs.org/api/esm.html#customizing-esm-specifier-resolution-algorithm
-        let tsNodeEsmLoader = `--experimental-specifier-resolution=node --loader ${tsNodeEsm}`
-
-        // starting in nodejs 20.19.0 and 22.7.0, the --experimental-detect-module option is now enabled by default.
-        // We need to disable it with the --no-experimental-detect-module flag.
-        // @see https://github.com/cypress-io/cypress/issues/30084
-        if (this.nodeVersion && (semver.gte(this.nodeVersion, '22.7.0') || semver.satisfies(this.nodeVersion, '>= 20.19.0 < 21.0.0'))) {
-          debug(`detected node version ${this.nodeVersion}, adding --no-experimental-detect-module option to child_process NODE_OPTIONS.`)
-          tsNodeEsmLoader = `${tsNodeEsmLoader} --no-experimental-detect-module`
-        }
-
-        // starting in nodejs 20.19.0 and 22.12.0, the --experimental-require-module option is now enabled by default.
-        // We need to disable it with the --no-experimental-require-module flag.
-        // @see https://github.com/cypress-io/cypress/issues/30715
-        if (this.nodeVersion && (semver.gte(this.nodeVersion, '22.12.0') || semver.satisfies(this.nodeVersion, '>= 20.19.0 < 21.0.0'))) {
-          debug(`detected node version ${this.nodeVersion}, adding --no-experimental-require-module option to child_process NODE_OPTIONS.`)
-          tsNodeEsmLoader = `${tsNodeEsmLoader} --no-experimental-require-module`
-        }
-
-        if (childOptions.env.NODE_OPTIONS) {
-          childOptions.env.NODE_OPTIONS += ` ${tsNodeEsmLoader}`
-        } else {
-          childOptions.env.NODE_OPTIONS = tsNodeEsmLoader
-        }
+
+      // TODO: get the tsconfig.json that applies to the users cypress.config.ts file
+      // right now, we are just using the tsconfig.json we find in the project root
+      const tsConfigIfExists = getTsconfig(this.projectRoot)
+
+      if (tsConfigIfExists) {
```

Comment:
Would not recommend having 'if' in the variable name here. Also, the 'exists' portion of the variable implies that this returns a boolean, but I see later that you're referencing '.path' off of it, so it's actually the tsconfig

```suggestion
      const tsConfig = getTsconfig(this.projectRoot)

      if (tsConfig) {
```

Reply:
addressed in 4f0a50f66f85cee14a5c5da284243ebc2b84b658